Assets/Scripts/Interactable.cs

```diff
@@ -11,21 +11,91 @@ public class Interactable : MonoBehaviour {
   [SerializeField] private Item giveItem = null;
   [SerializeField] private bool disableAfterFirstUse = false;
 
+  // Shaders for items
+  private Shader _regularShader;
+  private List<Shader> _childRegularShaders;
+  [SerializeField] private Shader _outlineShader;
+
+  // Renderers
+  private Renderer _rend;
+  private Renderer[] _childRends;
+
+  // Booleans
   private bool firstUse = true;
+  public bool shaderChanged = false;
+
+  // Prompt related variables
+  [SerializeField] private TextMesh Prompt;
+  public String promptText = "interact";
```

Comment:
Could this be serialized and private? ;p

Reply:
I left it public so that we can change the prompt text per interactable from the ui with ease. So the puddle we can change that to 'clean', and the mop and flowers to 'pick up'. Interact was something I thought we'd just have as a placeholder